tests/unit/test-util/test.logger.js

```diff
@@ -51,12 +54,12 @@ describe('logger', () => {
     it('lets you turn on verbose logging', () => {
       const log = new ConsoleStream({verbose: false});
       log.makeVerbose();
-      assert.equal(log.verbose, true);
+      sinon.assert.match(log.verbose, true);
     });
 
     it('logs names in verbose mode', () => {
       const log = new ConsoleStream({verbose: true});
-      assert.equal(
+      sinon.assert.match(
```

Comment:
This should also keep using `assert.equal()` because it should match the log message exactly